### ISSUE TITLE
Black on White accessibility concerns

### ISSUE THREAD

**Issue by camelPhonso** [enBloc-org/pathways#185]
### Pre-requisites

- [x] I have pulled the latest changes from the Origin
- [x] I have made sure to update my environment by running `npm install` on my terminal
- [x] I have tried to identify the source of the problem before submitting a report

### Description of the Bug

Currently black text is displayed directly on white background.

### What should happen instead

This could be ameliorated by setting global variables for `--black` and `--white` that display values slightly off from the full colour. This would improve accessibility.

### Assignment

This issue is free for anyone to take
